devito/operator.py

```diff
@@ -86,13 +87,12 @@ def __init__(self, expressions, **kwargs):
         self.dtype = retrieve_dtype(expressions)
         self.input, self.output, self.dimensions = retrieve_symbols(expressions)
         stencils = make_stencils(expressions)
-        self.offsets = {d.end_name: v for d, v in retrieve_offsets(stencils).items()}
 
         # Set the direction of time acoording to the given TimeAxis
         for time in [d for d in self.dimensions if d.is_Time]:
             if not time.is_Stepping:
                 time.reverse = time_axis == Backward
-
+        
```

Comment:
probably flake8 will bark

Reply:
You can safely assume that I will fix any flake issues before a merge since travis won't let me merge without it anyway. It would be more helpful if you could focus on things travis won't catch for now.